kernel/kstd/kstdio.cpp

```diff
@@ -143,7 +143,7 @@ bool panicked = false;
 	panicking = true;
 	tty->set_graphical(false);
 	printf("\033[41;97m\033[2J"); //Red BG, bright white FG
-	print("Good job, you crashed it.\nHere are the details, since you probably need them.\nTry not to mess it up again.\n\n");
+	print("Kernel panic!\n\nIf this occurs many times, then don't be afraid to report it on the\nGitHub repository.\n\n\nHere are the details, since you probably need them.\n");
```

Comment:
I think that adding information about submitting a GitHub issue is a great idea! While I would still like to have a little bit of character in the error message, maybe it could be a little nicer :) Maybe you could change it to this:

```suggestion
	print("Whoops! Something terrible happened.\nIf you weren't expecting this, feel free to open an issue on GitHub to report it.\nHere are the details:\n");
```